.env.example

```
@@ -0,0 +1,24 @@
+#   2. Start infrastructure: docker compose up -d postgres redis
+#   3. Push schema: pnpm db:push
+#   4. Run all apps: pnpm dev
+# =============================================================================
+
+# --- Shared: Database --------------------------------------------------------
+# PostgreSQL connection URL. Default works with the docker-compose.yml stack.
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/postgres?schema=public"
+
+# --- Shared: Redis -----------------------------------------------------------
+# Redis connection URL for EventBus pub/sub and BullMQ job queue.
+REDIS_URL="redis://localhost:6379"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)